SU2_CFD/src/solvers/CNEMONSSolver.cpp

```diff
@@ -495,7 +492,7 @@
        Note that we need to add a contribution for moving walls to the Jacobian. ---*/
       if (implicit) {
         /*--- Enforce the no-slip boundary condition in a strong way ---*/
-        for (iVar = nSpecies; iVar < nSpecies+nDim; iVar++) {
+        for (unsigned short iVar = nSpecies; iVar < nSpecies+nDim; iVar++) {
```

Comment:
@pcarruscag Any good ideas on how to resolve this?  I tried using auto, but that didnt work.  In my mind these types should be identical?

Reply:
It can be a false positive, if both nSpecies and nDim are unsigned short...
You can ignore it or static_cast<unsigned short>(nSpecies + nDim) but that's clunky so just ignore it if both nDim and nSpecies are the same type.